src/main/scala/resource/ExtractableManagedResource.scala

```diff
@@ -46,5 +46,7 @@ trait ExtractableManagedResource[+A] extends ManagedResource[A] {
    *        An either where the left hand side is the currently contained resource unless exceptions, in which case
    *        the right hand side will contain the sequence of throwable encountered.
    */
-  def either: Either[Seq[Throwable], A]
+  def either: ExtractedEither[Seq[Throwable], A]
 }
+
+case class ExtractedEither[+A, +B](either: Either[A, B])
```

Comment:
Right.  SO I think what I'd like to see (possibly not this CL, possibly before release) is that IF we're not going to use Either itself, then we create our own wrapper of Either that is its own datastrcture.

Also, you do not need to create an implicit conversion in the package object, simply do so in the companion object ( a better place).


Can you move `ExtractedEither` into its own file, and possibly rename it to `ArmResult`, so we know it's not exactly an either.  I'm ok with "easing" transition to Either, but FYI sometimes these "easy" mechanisms actually aren't as nice as you would think.  E.g. pattern matching MAY not work the way we want it (if the Left/Right pattern match is against `Any` and not `Either`).   Lots of subtle bugs can occur here.

So TL;DR;

* I'm ok NOT using supressed exceptions
* We should rename ExtractedEither into something more useful.
   - We can add explicit `.toTry`, `.toEither` methods to it
   - It's ok to break compatibility in the 2.0 release.  I'd rather do this than implicit conversion from `ExtractedEither => Either`, as this is too dangerous when we expect pattern matching.


Now the question is, since you've done all the work, would you like to flesh out a "ArmResult" (a.k.a. ExtractedEither) any further?